input/pagecontent/changes.md

```diff
@@ -6,5 +6,6 @@
 This change log documents the significant updates and resolutions implemented from version 0.1.0-ballot to TBD.
 
 #### Changes in this version
+- Simplified the FHIRPath for [AU eRequesting ServiceRequest Invariant](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/StructureDefinition-au-erequesting-servicerequest.html) au-ereq-srr-01 per [FHIR-47068](https://jira.hl7.org/browse/FHIR-47068)
```

Comment:
Please remove the change log entry for this fix, as it is QA-related and does not represent a substantive change.

Reply:
done